aztk/spark/client/cluster/helpers/get.py

```diff
@@ -0,0 +1,13 @@
+import azure.batch.models.batch_error as batch_error
+
+from aztk import error
+from aztk.spark import models
+from aztk.utils import helpers
+
+
+def get_cluster(spark_cluster_operations, cluster_id: str):
+    try:
+        pool, nodes = super(type(spark_cluster_operations), spark_cluster_operations).get(cluster_id)
```

Comment:
do you need to do that super(type thing?

Reply:
If not it fell into infinite recursion, but maybe there is a better way.

Reply:
spark_cluster_operations.get() would get an infinite loop?